src/tools/OptionsSchema.json

```diff
@@ -0,0 +1,398 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
```

Comment:
Should I add an "_comment": field? 

http://stackoverflow.com/questions/244777/can-comments-be-used-in-json